extensions/_beacons/http_status.py

```diff
@@ -105,30 +105,36 @@ def beacon(config):
                 r = requests.get(url, timeout=sites_config['timeout'])
             else:
                 r = requests.get(url, timeout=30)
+        # r.status_code
         except requests.exceptions.RequestException as e:
-            log.debug("Request failed: ", e)
+            log.debug("Request failed: %s", e)
+        if r.status_code >= 500:
+            log.debug('Response from status endpoint was invalid: '
+                      '%s', r.status_code)
         for json_response_item in sites_config['json_response']:
```

Comment:
Similar to below, this should be 
```suggestion
         for json_response_item in sites_config.get('json_response', []):
```